src/dbnode/integration/fs_bootstrap_index_test.go

```diff
@@ -168,6 +169,16 @@ func testFilesystemBootstrapIndexWithIndexingEnabled(
 
 	// Stop the server
 	defer func() {
+		parentDir := fmt.Sprintf("%s/data/", setup.FilePathPrefix())
+		openFilesBefore := countOpenDataFiles(parentDir)
+		require.NotZero(t, openFilesBefore)
+
+		require.NoError(t, setup.DB().Close())
+
+		openFilesAfter := countOpenDataFiles(parentDir)
+		require.Zero(t, openFilesAfter)
+
+		setup.Close()
```

Comment:
Should we make this a method we call on the `testSetup`? 

Or better yet we could add it just to the start of the `StopServer` call perhaps? Or add a new method `StopServerAndVerifyOpenFilesAreClosed()` which does this prework before stopping the server and can be shared between tests?

Reply:
Agree, added new method.